src/Core/Components/NavMenu/FluentNavBase.cs

```diff
@@ -29,12 +29,20 @@ public abstract class FluentNavBase : FluentComponentBase
     public Icon? Icon { get; set; }
 
     /// <summary>
-    /// Gets or sets the color of the icon. 
+    /// Gets or sets the color of the icon.
     /// It supports the theme colors, default value uses the themes drawer icon color.
     /// </summary>
     [Parameter]
     public Color IconColor { get; set; } = Color.Accent;
 
+    /// <summary>
+    /// Gets or sets the icon drawing and fill color to a custom value.
+    /// Needs to be formatted as an HTML hex color string (#rrggbb or #rgb) or CSS variable.
+    /// ⚠️ Only available when Color is set to Color.Custom.
+    /// </summary>
+    [Parameter]
+    public string? CustomColor { get; set; }
```

Comment:
I think the name `IconCustomColor` would be more appropriate in this case. To remain consistent with the `IconColor` property.